portfolio/src/main/webapp/script.js

```diff
@@ -13,20 +13,15 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Show/hide the description inside the element which has title as id.
  */
-function addRandomGreeting() {
-  const greetings = [
-    'I play tennis.',
-    '"An office is a place where dreams come true." — Michael Scott (The Office).',
-    '"A bug is never just a mistake. It represents something bigger. An error of thinking that makes you who you are." - Elliot (Mr Robot).',
-    'I have three dogs and one cat.'
-  ];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function toggleContent(title) {
```

Comment:
This is already quite cool and it works! Well done :)
Some disadvantages about your current approach:
 - you're manipulating style attributes directly which results in a mix of styling from CSS and style attributes and might be hard to understand in larger projects.
 - you need to know that there is always an element with a  "descrption" class as a child to hide/show. Also technically it could be far down in the DOM tree while you probably really  only want to support it if it's a direct descendant of the section.
 - you have to mix lookup by class name and id which might be confusing.

An alternative approach you could consider:
 - make your collapsible section a more generic pattern, e.g. always give your section a `section` class together with the id to identify it
 - Support another class `collapsed` (or `expanded`) on the section element which hides the child with the `description` class if present via CSS
 - in your JS, just add or remove that class directly from the element you get via the id lookup.

What do you think?

Reply:
note that all of this is just a suggestion. I'm completely fine if you want to leave it as is but wanted to start a discussion here.

Reply:
These were also some of my concerns.

- I thought about adding a section class, and since I am not using it for the style, I decided to not add it, but it might help with comprehensibility
- I hadn't considered using a collapsed class, and I think it's a good idea.